src/main/resources/messages.properties

```diff
@@ -1,27 +1,27 @@
-typeMismatch.courseConfigurationForm.totalClassMinutes = Must be a valid number.
-Min.courseConfigurationForm.totalClassMinutes = Must be greater than 0.
+typeMismatch.courseConfigurationForm.totalClassMinutes=Must be a valid number.
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge keys "typeMismatch.courseConfigurationForm.defaultMinutesPerSession, typeMismatch.courseConfigurationForm.totalClassMinutes, typeMismatch.makeupForm.entries.minutesMadeUp, typeMismatch.rosterForm.sectionModels.attendances.minutesMissed" that have the same value "Must be a valid number.". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aduplicated-values)